### DISCUSSION TITLE
Projects beta: Convert issue back to draft

### DISCUSSION THREAD

**Discussion by jfritz** [community/community#10151] [Projects and Issues] [answered]
I would appreciate a way to convert an issue back to "draft issue" state. I can add a repository column and bring an issue out of draft + move it to a repo by changing the "Repository" column. 

Is there a way to null out the repository column to bring an issue back to draft state (and moving it back under the parent organization out of the repo)?

**Comment by JackHowa** [comment]
Same! I'm seeing this problem in Sept 15, 2022.

**Comment by heptakod** [comment]
I need this feature too.  Currently, I'm deleting the issue if I mistakenly assign it a repo. There is no other way to erase or edit assinged repo.

**Comment by lucasfariaslf** [comment]
Really inconvenient that this is not possible out of the box, and that the issue is here opened for almost a year.

**Comment by simonebeets** [comment]
This functionality would definitely be useful.

**Comment by Nana2929** [comment]
Please I need this feature as well. Deeply need it if about to copy a project. All cards converted to issue are not copied and it causes so many troubles and pains.

**Comment by giolvani** [comment]
I am waiting for this feature as well...

**Comment by d108** [comment]
I wholeheartedly agree having the ability to revert an issue back to a draft could significantly enhance our workflow management.

**Comment by endodod** [comment]
Either I was blind or it's still not possible...

**Comment by edsonh** [comment]
Same happened to me today

**Comment by FWBlueWizard** [comment]
Still not a feature...

**Comment by ghost** [comment]
Absolutely! Would appreciate if could reverse the conversion, _made by mistake_, back, hopefully this year?

**Comment by erdogmush** [comment]
Same here. This feature is needed, not necessarily covert back to draft, but recreate a draft from a copy of it. It feels like this is reasonably easy to implement.

**Comment by Prime1Code** [comment]
push

**Comment by Grisgram** [comment]
I am also in desperate need of back-to-draft

**Comment by ric-h** [comment]
ping 🔝

**Comment by acannon-ccl** [comment]
Would definitely help with cleaning up!

**Comment by Rob-Ashworth** [comment]
I came across this problem too.  Only "fix" is to delete it and start again

**Comment by cgarren** [comment, marked as answer]
I would also love for this to be possible. It's a huge annoyance when you accidentally create an issue from an issue draft or change your mind

**Reply by eliezra236** [reply]
> I would also love for this to be possible. It's a huge annoyance when you accidentally create an issue from an issue draft or change your mind

I am not sure why this is marked as the answer. But it's not implemented yet, and I could also use it.

When copying project it's only possible to copy the draft issues, so if I want to copy a project and all it's issues it's only possible if all of the issues are draft issues

**Comment by jfritz** [comment]
I'm selecting an answer that's not an answer because github doesn't really care to respond here. This isn't a product-first product development product. It helps with organizing cards on a board and Github doesn't care about product owner features.

**Comment by angu-voy** [comment]
Please implement this.

**Comment by KristofaJosh** [comment]
I mean, this has been since 2022, can this be implemented already?

**Comment by kulterryan** [comment]
this isn't implemented yet!
